src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/PartyDetailsContentProvider.java

```diff
@@ -68,6 +69,9 @@ public PartyDetailsContent     createContent(
 
         boolean nameAmended = !oppositeParty.getName().equals(ownParty.getName());
         boolean addressAmended = !oppositeParty.getAddress().equals(ownParty.getAddress());
+        boolean phoneAmended = oppositeParty.getPhone().isPresent()
+            && !(oppositeParty.getPhone().orElse("")
+            .equalsIgnoreCase(ownParty.getMobilePhone().orElse("")));
```

Comment:
The `isPresent()` check isn't needed

Reply:
its needed, as we check if it was provided by opposite party and then changed by self